proto/umee/leverage/v1/query.proto

```diff
@@ -335,19 +151,84 @@ message QueryMarketSummaryResponse {
+  repeated cosmos.base.v1beta1.Coin supplied = 1 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+  // Collateral contains all uTokens the account has collateralized. It is denominated in uTokens, so both exponent and uToken exchange rate from each coin's market_summary must be applied to convert to base token symbol denom.
+  repeated cosmos.base.v1beta1.Coin collateral = 2 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+  // Borrowed contains all tokens the account has borrowed, including interest owed. It is denominated in base tokens, so exponent from each coin's registered_tokens entry must be applied to convert to symbol denom.
+  repeated cosmos.base.v1beta1.Coin borrowed = 3 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+}
+
+// QueryAccountHealth defines the request structure for the AccountHealth gRPC service handler.
+message QueryAccountHealth {
+  string address = 1;
+}
+
+// QueryAccountHealthResponse defines the response structure for the AccountHealth gRPC service handler.
+message QueryAccountHealthResponse {
+  // Supplied Value is the sum of the USD value of all tokens the account has supplied, includng interest earned.
+  string supplied_value = 1 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // Collateral Value is the sum of the USD value of all uTokens the account has collateralized.
+  string collateral_value = 2 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // Borrowed Value is the sum of the USD value of all tokens the account has borrowed, including interest owed.
+  string borrowed_value = 3 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // Borrow Limit is the maximum Borrowed Value the account is allowed to reach through direct borrowing.
+  string borrow_limit = 4 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // Liquidation Threshold is the Borrowed Value at which the account becomes eligible for liquidation.
+  string liquidation_threshold = 5 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+}
+
+// QueryLiquidationTargets defines the request structure for the
+// LiquidationTargets gRPC service handler.
+message QueryLiquidationTargets {}
+
+// QueryLiquidationTargetsResponse defines the response structure for the
+// LiquidationTargets gRPC service handler.
+message QueryLiquidationTargetsResponse {
+  repeated string targets = 1;
```

Comment:
please add a comment. What this string contains?

Reply:
This query is unchanged in this PR - but it's a list of addresses eligible for liquidation